src/Context/SoapContext.php

```diff
@@ -273,4 +273,15 @@ public function beforeStepCheckForException(BeforeStepScope $scope)
             throw $this->fault;
         }
     }
+
+    /**
+     * @Given I register the following headers:
+     */
+    public function iRegisterHeaders(TableNode $headers)
+    {
+        foreach ($headers->getRowsHash() as $key => $value) {
+            $soapHeaders[] = new \SoapHeader($key, $key, $value);
```

Comment:
We're using $key as $namespace param. What happens if we need a custom namespace? I'm ok to add this support later I guess.